imagescan/delta.go

```diff
@@ -293,6 +293,35 @@ func (d *deltaState) findBestNode(nodeNames []string, requiredMemory *inf.Dec, r
 	return candidates[0].name, nil
 }
 
+func (d *deltaState) nodeNum() int {
```

Comment:
Maybe `nodeCount` will sound more clear. NodeNum sounds like some index func.

Reply:
Why is this needed 🤔 since we use value outside the function, the lock is not effective here?